browserSetup.sh

```diff
@@ -6,7 +6,8 @@
 # you can easily debug this script with 'bash -x browserInstall.sh', it 
 # will show all commands then
 
-set -u -e -o pipefail # fail on unset vars and all errors, also in pipes
+# fail on unset vars and all errors, also in pipes
+set -beEu -o pipefail
```

Comment:
Seems like these extra options would be a good addition to the main setup script.